### PR TITLE
DESENG-452 - Refactoring migrations

### PR DESCRIPTION
**Pull request by ratheesh-aot**
Issue #: https://github.com/bcgov/met-public/issues/

*Description of changes:*

- Deleted old migration files
- Created [**ec0128056a33_table_until_feb_09_2024.py**](https://github.com/bcgov/met-public/commit/560888e024573478e566694272176ee84d0dda3a#diff-1ed7855700d6ea03eb587bd0d4f8b626125073a58bd138837dd9da0fdb9e6f52) with all tables until feb 09 2024
- Created [**37176ea4708d_data_until_feb_09_2024.py** ](https://github.com/bcgov/met-public/commit/560888e024573478e566694272176ee84d0dda3a#diff-5638e218df00520757a9e787a27a1fc54125437037aec9c3e49dc8fca7af8897) with all initial data until feb 09 2024

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-api/tests/utilities/factory_scenarios.py
Comment (NatSquared) [inline met-api/tests/utilities/factory_scenarios.py:1]: What's the reason for this change?

Comment (ratheesh-aot) [reply met-api/tests/utilities/factory_scenarios.py:1]: image

@NatSquared  in Model, the max length of component_id for comment is 10. So I shorted the text as the unit test failed.
